charts/cortx/README.md

```diff
@@ -154,6 +156,8 @@ helm uninstall cortx
 | rbac.create | bool | `true` | Enable the creation of RBAC resources, Role and RoleBinding, for the CORTX ServiceAccount. |
 | server.auth.adminAccessKey | string | `"cortx-admin"` | The admin user's Access Key |
 | server.auth.adminUser | string | `"cortx-user"` | Name of the admin user that is created when initializing the cluster |
+| server.certificateSecret | string | `""` | Name of Secret that contains the certificate |
+| server.certificateSecretKey | string | `""` | Name of Secret key that contains the certificate |
```

Comment:
Is it really necessary to provide this flexibility?

Say one day we can upgrade to using a real tlsSecret, those fields are fixed anyways. Why not just require a fixed key name, and just document that.